### PR TITLE
Replace Travis with GitHub Actions

### LINKED ISSUE #55: Switch to GitHub Actions
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all plugins except Pressbooks and this plugin deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/), the latest version of [this plugin](http://github.com/pressbooks/pressbooks-cas-sso/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks-cas-sso) to see if your bug or enhancement is already listed?

### Description

This repository is still using Travis, and should be switched to GitHub Actions. Noticed while working on #54.

### Steps to Reproduce

Not applicable.

### PR DESCRIPTION
**Pull request by SteelWagstaff**
This PR replaces Travis with GitHub Actions and adds Pressbooks Build Tools as a dev dependency (see https://github.com/pressbooks/pressbooks/issues/2235). It also makes some modifications to the package.json scripts to be compatible with Laravel Mix 6. Fix for https://github.com/pressbooks/pressbooks-cas-sso/issues/55 and partial fix for https://github.com/pressbooks/private/issues/466

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-    "build:production": "npm run production",
-    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "test": "npm run lint",
-    "lint": "npm run -s lint:scripts && npm run -s lint:styles",
-    "lint:scripts": "cross-env NODE_ENV=development node_modules/eslint/bin/eslint.js \"assets/src/scripts/*.js\"",
-    "lint:styles": "cross-env NODE_ENV=development node_modules/stylelint/bin/stylelint.js \"assets/src/styles/**/*.scss\" --syntax scss"
-  },
+	"scripts": {
+		"watch": "mix watch",
+		"prod": "npm run production",
```
Comment (arzola) [inline package.json]: I feel we don't need two production commands I think we can get rid of "prod" one

**Review by arzola — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (SteelWagstaff) [reply package.json]: Agree. I removed the 'prod' command.

**Review by arzola — APPROVED** [review]

**Comment by arzola** [comment]
Looks good, Tests are passing nice work Steel

**Comment by SteelWagstaff** [comment]
Thanks. Will you squash & merge and delete this branch @arzola ?

**Pull request merged**